content/patterns/radio/examples/radio.html

```diff
@@ -25,120 +25,121 @@
-            Note: The explicit setting of <code>forced-color-adjust</code> is necessary because some browsers do not use <code>auto</code> as the default value for SVG graphics.
+            CSS ファイル内のインライン SVG ラジオボタングラフィックが、ハイコントラスト設定でテキストコンテンツの色が変更されたときに背景と十分なコントラストを持つことを保証するために、SVG グラフィックの <code>forced-color-adjust</code> は <code>auto</code> に設定されます。
+            これにより、円要素の <code>stroke</code> 及び <code>fill</code> の色がハイコントラストの色設定によって上書きされます。
+            外側の <code>circle</code> の背景をハイコントラストの背景色と一致させるために、外側の <code>circle</code> の <code>fill-opacity</code> 属性はゼロに設定され、内側の <code>circle</code> の <code>stroke-opacity</code> 属性はゼロに設定されます。
+            <code>forced-color-adjust</code> を使用して <code>stroke</code> 及び <code>fill</code> プロパティに指定された色を上書きしない場合、これらの色はハイコントラストモードでも同じままであり、ラジオ円要素と背景のコントラストが不十分になったり、色がハイコントラストモードの背景と一致するとアイコンが見えなくなったりする可能性があります。<br>
+            注: 一部のブラウザでは SVG グラフィックのデフォルト値として <code>auto</code> を使用しないため、<code>forced-color-adjust</code> の明示的な設定が必要です。
           </li>
         </ul>
       </section>
 
       <section>
-        <h2 id="kbd_label">Keyboard Support</h2>
+        <h2 id="kbd_label">キーボードのサポート</h2>
         <table aria-labelledby="kbd_label" class="def">
           <thead>
             <tr>
-              <th>Key</th>
-              <th>Function</th>
+              <th>キー</th>
+              <th>機能</th>
             </tr>
           </thead>
           <tbody>
             <tr data-test-id="key-tab">
               <th><kbd>Tab</kbd></th>
               <td>
                 <ul>
-                  <li>Moves focus to the checked <code>radio</code> button in the <code>radiogroup</code>.</li>
-                  <li>If a <code>radio</code> button is not checked, focus moves to the first <code>radio</code> button in the group.</li>
+                  <li><code>radiogroup</code> 内のチェックされた <code>radio</code> ボタンにフォーカスを移動します。</li>
+                  <li><code>radio</code> ボタンがチェックされていない場合、フォーカスはグループの最初の <code>radio</code> ボタンに移動します。</li>
                 </ul>
               </td>
             </tr>
             <tr data-test-id="key-space">
-              <th><kbd>Space</kbd></th>
+              <th><kbd>スペース</kbd></th>
               <td>
                 <ul>
-                  <li>If the <code>radio</code> button with focus is not checked, changes the state to <code>checked</code>.</li>
-                  <li>Otherwise, does nothing.</li>
-                  <li>Note: The state where a radio is not checked only occurs on page load.</li>
+                  <li>フォーカスのある <code>radio</code> ボタンがチェックされていない場合、状態を <code>checked</code> に変更します。</li>
```

Comment:
これはARIAステートのことか……？自信ないです……
```suggestion
                  <li>フォーカスのある <code>radio</code> ボタンがチェックされていない場合、ステートを <code>checked</code> に変更します。</li>
```